File: Assignment__19/Assignment_3.c
#include<stdio.h>
#include<stdlib.h>
#include<stdbool.h>

bool Check(int Arr[] , int iLength)
{
    int iCnt = 0, bFlag = false; 
    for(iCnt = 0 ; iCnt < iLength ;iCnt++)
    {
        if(Arr[iCnt] == 11)
        {
            bFlag = 1;
            break;
        }
    }
    if(bFlag == 1)  
    {
        return true;
    }
    else{
        return false;
    }
}
int main()
{
    int iSize = 0 , iCnt = 0 ;
    int *ptr = NULL;
    bool bRet = false;

    printf("Enter number of Elements :  ");
    scanf("%d",&iSize);

    ptr = (int *)malloc(iSize * sizeof(int));
    if(ptr == NULL)
    {
        printf("Unable to locate memory : \n");
        return -1;
    }

    printf("Enter %d numbers : \n",iSize);
    for(iCnt = 0  ; iCnt < iSize ; iCnt++)
    {
        printf("Enter %d numbers : ",iCnt+1);
        scanf("%d",&ptr[iCnt]);
    }
    bRet = Check(ptr,iSize);
    if(bRet == true)
    {
        printf("11 is present...\n");

    }
    else
    {
        printf("11 is Absent...\n");
    }
    free(ptr);

    return 0 ;
}

File: Assignment__19/Assignment_java/Assignment_3.java

import java.util.*;

class Contains11
{
    public boolean Check(int []Arr , int iLength)
    {
        boolean bFlag = false;
        for(int iCnt = 0 ; iCnt < iLength ; iCnt++)
        {
            if(Arr[iCnt] == 11 ) 
            {
                bFlag = true;
                break;
            }
        }
        System.out.println();

        return bFlag;
    }
}
class Assignment_3
{

    public static void main(String A [])
    {
        boolean bRet = false;

        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter Number : ");
        int iSize = sobj.nextInt();
    

        int []ptr = new int[iSize];

        System.out.println("Enter "+iSize+" Elements : ");

        for(int iCnt = 0 ; iCnt < iSize; iCnt++)
        {
            System.out.print("Enter elements "+ (iCnt + 1) +" : ");
            ptr[iCnt] = sobj.nextInt();
        }
        Contains11 cobj = new Contains11();
        bRet = cobj.Check(ptr , iSize);
        if(bRet == true)
        {
            System.out.println("11 is PRESENT...");
        }
        else 
        {
            System.out.println("11 is ABSENT...");   
        }

    }
}